spark/dl/pom.xml

```diff
@@ -142,6 +142,7 @@
                     <systemProperties>
                         <bigdl.disableCheckSysEnv>true</bigdl.disableCheckSysEnv>
                     </systemProperties>
+                    <argLine>-Xmx6g</argLine>
```

Comment:
I hesitated to increase it further, but could. Really it's quite a big test! @qiuxin2012 do you know how much memory worked for you? 8g?